hal/src/common/thumbv7em/sercom/v1/uart.rs

```diff
@@ -171,7 +171,6 @@ macro_rules! uart {
                             w.rxpo().bits(rxpo); // Uses pad 3 for rx
                             w.txpo().bits(txpo); // Uses pad 2 for tx (and pad 3 for xck)
 
-                            w.form().bits(0x00); // No parity
```

Comment:
Just out of curiosity, what's the reason for this line being removed?

Reply:
I think I removed it when getting the UART working. Its safe either way - the UART is reset, and `0x00` (no parity) is the default value. I can put it back if we want to minimise the diff.

Reply:
It's fine to leave as-is if it's reset to this value anyways. As I said, just curious.